src/core/auth/client/AuthClient.ts

```diff
@@ -63,18 +63,23 @@ export class AuthClient {
     this.logout();
   }
 
-  private processAuthResponse(response: {
+  private async processAuthResponse(response: {
     token: string;
     refreshToken: string;
     jwtData: JWTData;
   }) {
-    currentUserAtom.setUser.dispatch({
+    // const profile = await this._apiClient.get<profileResponse>(
+    const profileUserdata = await this._apiClient.get('/users/current_user', {}, true);
+    const jwtUserdata = {
       username: response.jwtData.preferred_username,
       token: response.token,
       email: response.jwtData.email,
       firstName: response.jwtData.given_name,
       lastName: response.jwtData.family_name,
-    });
+    };
+    // @ts-expect-error - Fix me - load profile in better place
```

Comment:
```suggestion
    // @ts-expect-error - Fix me - load profile in better place and use 'type profileResponse' for describing response to remove error
```

Or may be there's a point to move 
![image](https://user-images.githubusercontent.com/50058726/197761157-109ad9c4-50c5-477d-8b84-d009092c3a99.png)
from the `features/user_profile` to core?